Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/Db.ts

```diff
@@ -12,13 +12,17 @@ import {
   W,
   ClientSession,
   ObjectId,
+  FindOptions,
+  ReplaceOptions,
 } from 'mongodb';
 import { Logger } from '@edfi//meadowlark-core';
 import { MeadowlarkDocument } from '../model/MeadowlarkDocument';
+import { AuthorizationClient } from '../model/AuthorizationClient';
 
 const MONGO_URL_DEFAULT = 'mongodb://mongo1:27017,mongo2:27018,mongo3:27019/?replicaSet=rs0';
 export const DATABASE_NAME = process.env.MEADOWLARK_DATABASE_NAME ?? 'meadowlark';
-export const COLLECTION_NAME = 'documents';
+export const DOCUMENT_COLLECTION_NAME = 'documents';
+export const AUTHORIZATION_COLLECTION_NAME = 'authorizations';
```

Comment:
Chose "authorizations" as table/collection name - looking at putting all authorization server objects in this collection - will mirror in postgresql implementation.